app/controllers/reports_controller.rb

```diff
@@ -31,7 +37,10 @@ def report_params
     development_params = (1..ReportForm::DEVELOPMENT_OBJECTIVES).map do |n|
       "development_#{n}"
```

Comment:
With the development objectives it may work better. But with the SMART I'm not sure, cause they also have 'what' and 'how'. I wanted it to be unified for those kinds.